exa/core/container.py

```diff
@@ -32,6 +33,12 @@ class Container(object):
     _getter_prefix = 'compute'
     _cardinal = None    # Name of the cardinal data table
 
+    @property
+    def log(self):
+        name = '.'.join([self.__module__,
+                         self.__class__.__name__])
+        return logging.getLogger(name)
```

Comment:
This approach is really nice. It makes it very easy to track down where a log message occurs.